### ISSUE TITLE
[Bug]: Water is grey

### ISSUE THREAD

**Issue by SkyKingPX** [TeamJM/journeymap#584]
### What happened?

Before yesterday all was good, but then I join my server and the water on the map gets grey. What is this?

I AM USING VERSION 5.9.5

### Minecraft Version

1.19.3

### Version

5.9.4 (Default)

**Comment by SkyKingPX** [comment]
1. [2023-04-13-3.log](https://github.com/TeamJM/journeymap/files/11233193/2023-04-13-3.log)
2. [journeymap.log](https://github.com/TeamJM/journeymap/files/11233201/journeymap.log)

And no, I didn´t change anything

@mysticdrew

**Comment by mysticdrew** [comment]
If you create a singleplayer world in that same instance, does it do it in the singleplayer world too? Or only in the multiplayer world?

**Comment by SkyKingPX** [comment]
Also in Singleplayer:
![image](https://user-images.githubusercontent.com/95442431/232066429-3d761855-e174-44b9-856e-0459499556bf.png)

@mysticdrew

**Comment by mysticdrew** [comment]
I suspect WorldEdit. 
I would try removing all your mods other than fabric api and journeymap and testing your singleplayer world, see if that still has the issue.

**Comment by SkyKingPX** [comment]
Yes, now it is working, but I have to use WorldEdit. How I can still use WorldEdit? @mysticdrew

**Comment by SkyKingPX** [comment]
Is it fixed now??? @mysticdrew

**Comment by mysticdrew** [comment]
is there a fixed label on this issue?

**Comment by SkyKingPX** [comment]
No, but no one is saying something about this bug

**Comment by mysticdrew** [comment]
It will get fixed when I get time to fix it. Be patient, this is not a job, I do this when I have free time.

**Comment by SkyKingPX** [comment]
Ook

**Comment by mysticdrew** [comment]
This is the last bug I want to fix before I release 5.9.7 today. So the faster we can figure this out the better.

**Comment by SkyKingPX** [comment]
Ok

**Comment by mysticdrew** [comment]
Any luck?
I am going to be releasing 5.9.7 tonight. Which will be the last release until 1.20 all support for 1.18 and 1.19 versions will be dropped.

**Comment by mysticdrew** [comment]
These are the mods I am running. 
![image](https://github.com/TeamJM/journeymap/assets/5459941/9a9839b6-398e-4a4a-97a4-7bebb4aace63)
Water is not grey. 
Am I missing something?

**Comment by mysticdrew** [comment]
ahh ok I so was able to replicate it. 
Unchecking 
Show Water Biome Colors makes the water grey.
![image](https://github.com/TeamJM/journeymap/assets/5459941/229b04cc-6c9e-4713-9660-3040bf4b96f3)

**Comment by SkyKingPX** [comment]
Ok, but I didn’t change it

**Comment by mysticdrew** [comment]
It is essentials+carpet causing the issue. 
Show biome colors is off by default which is why it was showing grey for you. If you enable it they will display correctly. I am looking into the issue now with essentials.

**Comment by SkyKingPX** [comment]
Ok thanks

**Comment by mysticdrew** [comment]
This is a bug in EssentialsClient
I made an issue. 
https://github.com/senseiwells/EssentialClient/issues/113

**Comment by mysticdrew** [comment]
I will be providing a workaround for this issue on my end. But that bug may cause issues with other mods that get water colors.

**Comment by mysticdrew** [comment]
5.9.7 is on curseforge and modrinth with the fix
